framework/rbac/DbManager.php

```diff
@@ -80,6 +80,10 @@ class DbManager extends BaseManager
      */
     public $cache;
     /**
+     * @var bool should assignments be cached
+     */
+    public $cacheAssignments = false;
```

Comment:
We've settled that this is dangerous for long running processes and is questinable when there's a large number of assignments (users). It might require to be documented elsewere and possibly the base applications config should be updated.